docs/guide/pagination.md

```diff
@@ -159,6 +159,77 @@ query {
 > OffsetPaginated follow the same rules for the default pagination limit, and can be configured
 > in the same way as explained above.
 
+### Customizing queryset resolver
+
+It is possible to define a custom resolver for the queryset to either provide a custom
+queryset for it, or even to receive extra arguments alongside the pagination arguments.
+
+Suppose we want to pre-filter a queryset of fruits for only available ones,
+while also adding [ordering](./ordering.md) to it. This can be achieved like:
+
+```python title="types.py"
+
+@strawberry_django.type(models.Fruit)
+class Fruit:
+    name: auto
+    price: auto
+
+
+@strawberry_django.order(models.Fruit)
+class FruitOrder:
+    name: auto
+    price: auto
+
+
+@strawberry.type
+class Query:
+    @straberry.offset_paginated(OffsetPaginated[Fruit], order=order)
+    def fruits(self, only_available: bool = True) -> QuerySet[Fruit]:
+        queryset = models.Fruit.objects.all()
+        if only_available:
+            queryset = queryset.filter(available=True)
+
+        return queryset
+```
+
+This would produce the following schema:
```

Comment:
I'm a huge fan of adding the resulting schema to the docs for each of these things, it's been a big point of frustration for me with the docs not seeing how it should come out.

Reply:
Thanks for the feedback! I'm planing on doing some improvements to the docs soon and I'll do more of this!